test/models/event_search_test.rb

```diff
@@ -0,0 +1,20 @@
+require 'test_helper'
+
+class EventSearchTest < ActiveSupport::TestCase
+
+  # ArgumentError: wrong number of arguments (given 1, expected 0)
```

Comment:
Matt, I may need a hand here, I'm getting this error but it doesn't seem to make sense since the event:98 should expect arguments